ansible_base/lib/abstract_models/common.py

```diff
@@ -66,27 +81,46 @@ def _attributable_user(self, warn_nonexistent_system_user):
                     user = get_user_model().objects.get(username=system_username)
                 except get_user_model().DoesNotExist:
                     if warn_nonexistent_system_user:
-                        logger.warn(f"SYSTEM_USERNAME is set to {system_username} but no user with that username exists. User attribution will be None.")
+                        logger.warn(f"SYSTEM_USERNAME is set to {system_username} but no user with that username exists.")
```

Comment:
I think `warn` is deprecated, `warning` should be used

Reply:
Yea, this was actually biting me in the tests yesterday because I was trying to test a `warning` log but using the logic that was used to test this `warn`. I'll update this reference.